opts/mount.go

```diff
@@ -102,6 +111,20 @@ func (m *MountOpt) Set(value string) error {
 				volumeOptions().DriverConfig.Options = make(map[string]string)
 			}
 			setValueOnMap(volumeOptions().DriverConfig.Options, value)
+		case "tmpfs-size":
+			sizeBytes, err := units.RAMInBytes(value)
+			if err != nil {
+				return fmt.Errorf("invalid value for %s: %s", key, value)
+			}
+			tmpfsOptions().SizeBytes = sizeBytes
+		case "tmpfs-mode":
+			var mode os.FileMode
+			if ui64, err := strconv.ParseUint(value, 0, 32); err != nil {
```

Comment:
No.
Updated PR to use `ParseUint(value, 8, ..)` instead.
Linux tmpfs uses `strtoul(..,8,.. )` as well: http://lxr.free-electrons.com/source/mm/shmem.c#L3440